build-system/tasks/helpers.js

```diff
@@ -19,7 +19,7 @@ const babelify = require('babelify');
 const browserify = require('browserify');
 const buffer = require('vinyl-buffer');
 const colors = require('ansi-colors');
-const conf = require('../compile/build.conf');
+const conf = require('../../babel.config');
```

Comment:
@jridgewell Based on [this](https://babeljs.io/docs/en/config-files#supported-file-extensions) documentation, I think we should eventually eliminate the `require`-ing of `babel.config.js` and refactor code in files like this. WDYT?

Reply:
I agree with this; it was odd to see. That being said, I think that's why `build.conf.js` may have existed separately in the first place: to define those shared methods somewhere for other files to use. It wasn't clear how much boy-scouting was desired here, or if the goal was to just combine them, so I went conservative the first time around. I'd be happy to refactor/clean this up a bit; will defer to your judgement on how much of that should happen here vs. in a follow-up refactor PR.